Tests/SwiftProtobufTests/Test_Map.swift

```diff
@@ -68,6 +68,7 @@ class Test_Map: XCTestCase, PBTestHelpers {
         assertDecodeSucceeds([10, 4, 8, 1, 16, 2]) {
             $0.mapInt32Int32 == [1: 2]
         }
+        // TODO: Why should this fail?  It appears to actually be valid.
```

Comment:
After making the consumed changed, this test starts failing.  The comment says it is an invalid bad wire type, but I think it actually is a field 1 startGroup.  Where it gets interesting is I don't see our unknown handling property dealing with this.  Our generated `messageDecode` is:

```swift
  mutating func _protoc_generated_decodeMessage<D: SwiftProtobuf.Decoder>(decoder: inout D) throws {
    try withExtendedLifetime(_storage) { (_storage: _StorageClass) in
      while let fieldNumber = try decoder.nextFieldNumber() {
        try decodeField(decoder: &decoder, fieldNumber: fieldNumber)
      }
    }
  }
```

But how do groups factor into this.  Isn't this going to say it is field 1, but never really capture that it it was a startGroup and there *has* to be an endGroup marker and all sub data could into this messages unknowns.